### DISCUSSION TITLE
Not able to upload a CSV file

### DISCUSSION THREAD

**Discussion by jugalb-icpl** [DefectDojo/django-DefectDojo#11100] [Q&A] [answered]
I am facing a problem uploading a report
- Not able to upload below file, I have selected Qualys scan
- it shows "No findings were added/updated/closed/reactivated as the findings in Defect Dojo are identical to those in the uploaded report." but currently system has no other findings

[ASV_test-2.csv](https://github.com/user-attachments/files/17457350/ASV_test-2.csv)

please help me here, Thanks in advance

**Comment by manuel-sommer** [comment, marked as answer]
Hi @jugalb-icpl ,
I had a quick look. The present fields in your file look very different than the unittest files from the implement Qualys parser. 
You can take a look here and compare the fields from the unittests with the fields in your file: https://github.com/DefectDojo/django-DefectDojo/tree/master/unittests/scans/qualys

Either you selected the wrong parser or the Qualys output changed. Then, you can open up an issue and Qualys can be fixed through a PR. 

Does this help?

Best,
Manuel

**Comment by jugalb-icpl** [comment]
Thank you so much for response, Just a quick question, can I select some other scan type for this file ?

**Reply by manuel-sommer** [reply]
Hi @jugalb-icpl , what scanner did you use to produce this outputfile? If it was Qualys, I don't expect any other parser being able to parse this file. Furthermore, I don't recommend trying this as the other parser could evolve over time and will not support this file structure anymore. Summarizing: I would recommend to select the right scan type and if the parser is not good enough yet, open up an issue.
